### ISSUE TITLE
CI: v1.8: Suite-k8s-1.11.K8sVerifier Runs the kernel verifier against Cilium's BPF datapath

### ISSUE THREAD

**Issue by tklauser** [cilium/cilium#14071]
```
/home/jenkins/workspace/Cilium-PR-K8s-1.11-net-next/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:461
test-verifier pod not ready after timeout
Expected
    <*errors.errorString | 0xc0003dd6a0>: {
        s: "timed out waiting for pods with filter test-verifier to be ready: 4m0s timeout expired",
    }
to be nil
/home/jenkins/workspace/Cilium-PR-K8s-1.11-net-next/src/github.com/cilium/cilium/test/k8sT/Verifier.go:58
```

Seen on #14015: https://jenkins.cilium.io/job/Cilium-PR-K8s-1.11-net-next/1473/ - [test_results_Cilium-PR-K8s-1.11-net-next_1473_BDD-Test-PR.zip](https://github.com/cilium/cilium/files/5561200/test_results_Cilium-PR-K8s-1.11-net-next_1473_BDD-Test-PR.zip)

Also occured on `v1.8` upstream branch at least two times:

https://jenkins.cilium.io/view/Cilium-v1.8/job/cilium-v1.8-k8s-1.11-kernel-net-next/726/ - [test_results_cilium-v1.8-k8s-1.11-kernel-net-next_726_BDD-Test-PR.zip](https://github.com/cilium/cilium/files/5561232/test_results_cilium-v1.8-k8s-1.11-kernel-net-next_726_BDD-Test-PR.zip)
https://jenkins.cilium.io/view/Cilium-v1.8/job/cilium-v1.8-k8s-1.11-kernel-net-next/722/ - [test_results_cilium-v1.8-k8s-1.11-kernel-net-next_722_BDD-Test-PR.zip](https://github.com/cilium/cilium/files/5561195/test_results_cilium-v1.8-k8s-1.11-kernel-net-next_722_BDD-Test-PR.zip)

/cc @pchaigno

**Comment by pchaigno** [comment]
I think I know the fix. We need to extend the tolerations for the `test-verifier` pod to include `node.kubernetes.io/unreachable`.

**Comment by pchaigno** [comment]
@tklauser Not 100% sure on the fix in #14074, so worth keeping an eye for this flake in case we need to reopen.

**Comment by tklauser** [comment]
@pchaigno Thanks, will help to keep an eye on it.
